adding_species_manuscript.tex

```diff
@@ -653,7 +671,7 @@ \subsection*{Extracting model parameters from the literature}
 \subsection*{Filling out the missing pieces}
 
 For some species it may be difficult to obtain citeable values for the necessary model parameters, even when combining different sources. We provide several suggestions for dealing with this scenario (see Table \ref{tab:param-mod}).
-If the species of interest does not have citeable mutation or recombination rates, the average genome-wide rates published for a closely related species may be used. This may slightly skew the average genetic diversity and the average genetic linkage, but will still likely produce useful simulations. A similar approach can be applied to the DFE if simulating natural selection.
+If the species of interest does not have citeable mutation, recombination, or gene conversion rates, the average genome-wide rates published for a closely related species may be used. This may slightly skew the average genetic diversity and the average genetic linkage, but will still likely produce useful simulations. A similar approach can be applied to the DFE if simulating natural selection.
```

Comment:
I'm considering sticking with the original wording because I'm not sure that we want to put gene conversion in the same basket as recombination and mutation rates here. Mutation and recombination rates are required for each model. We don't require parameters for gene conversion for all models. So, one might add a species model without gene conversion parameters. You can't do this for mutation and recombination rates.

Reply:
Would it be valid to use gene conversion rates from a closely related species? (Or at least as valid as for mutation and recombination rates.) If yes, I think it should be made clear here as well, but perhaps also state explicitly that this is only if a gene conversion rate is required to model the important diversity in your organism.

Reply:
I'll try to clarify this point here by adding a separate sentence about gene conversion here. I don't want to lump it together with mutation and recombination rates, which are basic essentials for every species.